website/docs/r/kusto_cluster.html.markdown

```diff
@@ -104,6 +104,10 @@ A `virtual_network_configuration` block supports the following:
 
 * `data_management_public_ip_id` - (Required) Data management's service public IP address resource id.
 
+* `enabled` - (Optional) Whether the virtual network is enabled. Possible values are: `true` and `false`. Defaults to `true`.
+
+~> **NOTE:** Currently `enabled` supports changes only from `true` to `false` and `public_network_access_enabled` will also be set to `false` when `enabled` is set to `false`.
```

Comment:
The `subnet_id`, `engine_public_ip_id`, and `data_management_public_ip_id` fields need to have the text `Changing this forces a new Kusto Cluster to be created.` added to them since they are now `ForceNew`.

```suggestion
~> **NOTE:** The `enabled` field only supports changing the value from `true` to `false`. Changing the `enabled` field to `false` will cause the `public_network_access_enabled` field to be set to `false` as well.
```